python/xorbits/_mars/dataframe/datasource/read_csv.py

```diff
@@ -739,8 +739,25 @@ def read_csv(
     columns_value = parse_index(mini_df.columns, store_data=True)
     if index_col and not isinstance(index_col, int):
         index_col = list(mini_df.columns).index(index_col)
+
+    # convert path to abs_path
+    if isinstance(path, (list, tuple)):
```

Comment:
Did you have test for this situation?

Reply:
Yes, i have tested this situation.

Reply:
I mean the ut, which UT covered this branch?

Reply:
I added some UTs in current commit.